rpcs3/Emu/RSX/CgBinaryVertexProgram.cpp

```diff
@@ -35,7 +37,7 @@ std::string CgBinaryDisasm::GetMaskDisasm(bool is_sca)
 		if (d3.vec_writemask_w) ret += "w";
 	}
 
-	return ret.empty() || ret == "xyzw" ? "" : ("." + ret);
+	return ret.empty() || ret == f ? "" : ("." + ret);
```

Comment:
This is actually a regression in optimization, it was *not* constructing any std::string and comparing const char* with it instead. 
If you really want to optimize do:
```suggestion
	return ret.empty() || ret == "xyzw"sv ? "" : ("." + ret);
```

Reply:
f is constructed only once at the very beginning, I've done this to unify all methods

Reply:
std::string copies the characters to a buffer of itself, std::string_view does not.

Reply:
And in every other place there are checks with ```f```. Do you mean they all should be replaced with ```"xyzw"sv``` check

Reply:
Done, please check.